magni_bringup/scripts/launch_core.py

```diff
@@ -68,10 +70,26 @@ def get_conf():
         print "Error calling pifi"
         subprocess.call(["chronyc", "waitsync", "6"]) # Wait up to 60 seconds for chrony
 
+    boardRev = 0
```

Comment:
The firmware uses a value of 1 for unknown.   A zero would work here in firmware logic but I don't like using 0 because it is a common value that can get set on initialization and so on so 1 would be a more 'definite' value.  
We could use 1 OR could use 2 so firmware knows it was set to 2 by the host and thus the I2C was attempted to be used but failed.  This may be of some value in diagnose I2C issues.

Reply:
This value will not be written to the firmware unless we get a non-0 value anyway, so I don't think it is worth thinking it about too much.